.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
 node_js:
   - 6
+  - 7
```

Comment:
Node.js v7.x isn't maintained at the moment, it has reached the end-of-life period, according to https://github.com/nodejs/Release. There is no need to test on this version of Node.js.

Reply:
@belochub thanks, I removed `7.x` from other Metarhia libs